js/chart.js

```diff
@@ -1,22 +1,87 @@
-function Chart(){
-    var renderChart = function(){
-        //code to render charts
-    }
-    this.coloumnChart = function(){
+function Chart() {
+
+    this.coloumnChart = function () {
```

Comment:
Chart.js is a class which should contain only a render(type) method. From here u need to make a call to the respective Chart.